uwuipy/uwuipy.py

```diff
@@ -107,11 +109,12 @@ class uwuipy:
         "(ᴜ‿ᴜ✿)",
         "~(˘▾˘~)",
         "(｡ᴜ‿‿ᴜ｡)",
+        ">/////<",
     ]
 
     def __init__(
         self,
-        seed: int | None = None,
+        seed: Union[int, None] = None,
```

Comment:
From what I've seen, it's usually preferred to do `Optional[int]` rather than `Union[int, None]` if you're not going to use `|` syntax.

Reply:
I guess that Optional[int] better.